HW4/nataliia/t3.py

```diff
@@ -0,0 +1,14 @@
+n = int(input("Enter number: "))
```

Comment:
n=int(input('enter some number from fibonacci list='))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n}, {fibonacci_numbers}')